todo-app/src/Add-Todo.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import ListItem from "./List-Item";
+
+let todoItems = [];
```

Comment:
This is wrong. You are keeping track of todos in two separate places: `Main.js` (via `todos.json`) and `Add-Todo.js`, via this array. You basically have two lists of todos.

You need to keep track of your state in a single place. You should import `todos.json` into your `Main.js` component's state, and then add todos there by calling `this.setState`.